### ISSUE TITLE
Support for NFC

### ISSUE THREAD

**Issue by 3GDXC** [dotnet/maui#5153]
### Description

I would like to create a inventory management system using MAUI; warehouses currently use a mixture of Barcodes, QR codes, RFID and NFC tagged products and/or stillages.  It would be good if MAUI provided as part of the essentials namespace NFC support for devices.

### Public API Changes

var nfc = new NFC();
nfc.event+=processEvent;
await nfc.ListenAsync( );

 

### Intended Use-Case

holding the device up to/close to a NFC tag would trigger the event with the event args containing the NFC tag data

**Comment by 3GDXC** [comment]
Also support for RFID as above would be good

**Comment by jsuarezruiz** [comment]
> Any update on this issue. I also have requirement of using NFC in my Maui app instead many people have an eye on this. @Redth @jsuarezruiz

At this moment we are focused in performance and quality (bug fixing). We will take into account requests like this one in the future but for now, you can use the code from community plugins like https://github.com/franckbour/Plugin.NFC

**Comment by notyournick** [comment]
any news? has been a while. 

Happy New Year to all.

**Comment by Amine-Smahi** [comment]
News ?

**Comment by HeathersZen** [comment]
Bump!

**Comment by uestadeus** [comment]
Anything?

**Comment by developerSoluzione1** [comment]
I took an old Xamarin project and brought it to Maui with basically no changes. Tested in Android and works fine

https://github.com/acaliaro/XamarinDemoNfc

**Comment by uestadeus** [comment]
> I took an old Xamarin project and brought it to Maui with basically no changes. Tested in Android and works fine
> 
> https://github.com/acaliaro/XamarinDemoNfc

Also for IOS ?

**Comment by uestadeus** [comment]
Oh my mistake it s written there only for android. i missed that. thanks anyway

**Comment by acaliaro** [comment]
> > I took an old Xamarin project and brought it to Maui with basically no changes. Tested in Android and works fine
> > https://github.com/acaliaro/XamarinDemoNfc
> 
> Also for IOS ?

Yes the Xamarin sample was also for iOS but I have not tested in Maui ( I think it should works)

**Comment by Rigo09** [comment]
> Tomé un antiguo proyecto de Xamarin y lo traje a Maui básicamente sin cambios. Probado en Android y funciona bien
> 
> https://github.com/acaliaro/XamarinDemoNfc

Hola, me podrías ayudar por favor con los pasos que seguistes para traerlo a MAUI?

**Comment by acaliaro** [comment]
> > Tomé un antiguo proyecto de Xamarin y lo traje a Maui básicamente sin cambios. Probado en Android y funciona bien
> > https://github.com/acaliaro/XamarinDemoNfc
> 
> Hello, could you please help me with the steps you follow to bring it to MAUI?

https://github.com/acaliaro/MauiAppTestNfcAndroid

**Comment by andreas-spindler-mw** [comment]
Any updates?

The [Plugin.NFC](https://github.com/franckbour/Plugin.NFC) seems to be not working on newer iOS versions: https://github.com/franckbour/Plugin.NFC/issues/148

**Comment by kennykanp** [comment]
I've been running tests on both net7.0-ios and net8.0-ios, but I keep encountering this error
```
Time	Device Name	Type	PID	Tag	Message
Feb  3 18:42:18	Kennys-iPhones	Notice	29865	Xamarin.PreBuilt.iOS(CoreNFC)	-[NFCNDEFReaderSession beginSession]:363

```
```
Time	Device Name	Type	PID	Tag	Message
Feb  3 18:46:19	Kennys-iPhones	Notice	29865	Xamarin.PreBuilt.iOS(CoreNFC)	-[NFCNDEFReaderSession _invalidateSessionWithCode:message:finalUIState:activateCallback:]:971 code=4099, finalUIState=0, activateCallback=1
Time	Device Name	Type	PID	Tag	Message
Feb  3 18:46:19	Kennys-iPhones	Error	29865	Xamarin.PreBuilt.iOS(CoreNFC)	-[NFCNDEFReaderSession beginSessionWithConfig:]:443 error:Error Domain=NSCocoaErrorDomain Code=4099 UserInfo={NSDebugDescription=<private>}, errorCode: 0x1003
```

**Comment by Amine-Smahi** [comment]
Any update ?

**Comment by napangelo** [comment]
Hi, any news on the integration of nfc support in maui? is there an approximate date? @jsuarezruiz 

Thanks

**Comment by andru196** [comment]
Any update ?
